pkgs/by-name/ic/icecream-sundae/package.nix

```
@@ -0,0 +1,59 @@
+{
+  cmake,
+  fetchFromGitHub,
+  glib,
+  icecream,
+  lib,
+  libcap_ng,
+  libnl,
+  lzo,
+  meson,
+  ncurses,
+  ninja,
+  nix-update-script,
+  pkg-config,
+  stdenv,
+  zstd,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "icecream-sundae";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "JPEWdev";
+    repo = "icecream-sundae";
+    rev = "v${version}";
+
+    sha256 = "sha256-XFDVkPRwjbJa79lLMeMN78uuHGQRJ8rH10dKipCDjM8=";
+  };
+
+  passthru.updateScript = nix-update-script { };
+
+  buildInputs = [
+    ncurses
+    glib
+    libnl
+    icecream
+    lzo
+    libcap_ng
+    zstd
+  ];
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    cmake
+  ];
+  mesonFlags = [ ];
+
+  meta = with lib; {
+    description = "TUI for monitoring icecream clusters";
+    homepage = "https://github.com/JPEWdev/icecream-sundae/";
+    license = licenses.gpl2;
```

```suggestion
    license = licenses.gpl2Plus;
```
`licences.gpl#` (where # is 2 or 3) are deprecated (Nix values, not licences themselves), you now must specify `gpl#Only` or `gpl#Plus`. In this case, it's the latter: https://github.com/JPEWdev/icecream-sundae/blob/fadb8e9f9ca13ffe36c970b2101f481edfceae29/src/scheduler.cpp#L8